src/commands/scanner/run.ts

```diff
@@ -149,6 +153,10 @@ export default class Run extends ScannerRunCommand {
 			const pmdConfig = normalize(untildify(this.flags.pmdconfig as string));
 			options.set(CUSTOM_CONFIG.PmdConfig, pmdConfig);
 		}
+		
+		// Capturing verbose-violations flag value (used for RetireJS output)
+		options.set(CUSTOM_CONFIG.VerboseViolations, this.flags["verbose-violations"] ? "true" : "false")
```

Comment:
Since you changed `RetireJsEngine.ts` line 179 to use `.has()`, you'll want to wrap this in an `if` so it's only set if the flag is used.
Something like:
```
if (this.flags['verbose-violations']) {
   options.set(CUSTOM_CONFIG.VerboseViolations, 'true');
}
```

Reply:
Oh right, thanks!